docs/pages/guides/using-sentry.mdx

```diff
@@ -294,6 +294,16 @@ In order to ensure that errors are reported reliably, Sentry defers reporting th
 
 Unless `enableInExpoDevelopment: true` is set, all your dev/local errors will be ignored and only app releases will report errors to Sentry. You can call methods like `Sentry.Native.captureException(new Error('Oops!'))` but these methods will be no-op.
 
+## Troubleshooting
+
+<Collapsible summary="I'm seeing `error: project not found` in my build logs">
+
+This error is caused by the script that tries to upload sourcemaps during build.
```

Comment:
```suggestion
This error is caused by the script that tries to upload source maps during build.
```

Naming nit.

Reply:
The other 17 occurences on this page use 'sourcemaps', so I went with consistency. If 'source maps' is better, I can update all of them.

Reply:
I believe that `source map` is a correct name, so it would be nice if you can correct that. See:
* https://docs.sentry.io/platforms/javascript/sourcemaps/
* https://developer.chrome.com/blog/sourcemaps/
* https://firefox-source-docs.mozilla.org/devtools-user/debugger/how_to/use_a_source_map/index.html

Reply:
Yes, you are right, which is why I've already updated them :)